pkg/services/sqlstore/sqlstore.go

```diff
@@ -80,7 +80,7 @@ func (ss *SqlStore) Init() error {
 	x = engine
 	dialect = ss.Dialect
 
-	migrator := migrator.NewMigrator(x)
+	migrator := migrator.NewMigrator(engine)
```

Comment:
Use `engine` since it's a local variable, instead of `x`which is a global variable we want to get rid of.